cloud/docker/docker.py

```diff
@@ -542,7 +542,7 @@ def _human_to_bytes(number):
             return int(number[:-len(each)]) * (1024 ** i)
         i = i + 1
 
-    raise ValueError('Could not convert %s to integer' % (number,))
+    return int(number)
```

Comment:
This would still be a more accurate error message. But I could just trap the error and add some explanation? Right now you always get "Cannot convert 0 to integer" and it just doesn't work.

Ideally this should actually be handled in the calling function so the user knows which parameter they messed up.